docs/howto/operator/kubernetes.rst

```diff
@@ -22,150 +22,96 @@
 KubernetesPodOperator
 =====================
 
+The :class:`~airflow.providers.cncf.kubernetes.operators.kubernetes_pod.KubernetesPodOperator` allows
+you to create and run Pods on a Kubernetes cluster.
+
+.. contents::
+  :depth: 1
+  :local:
+
 .. note::
   If you use `Google Kubernetes Engine <https://cloud.google.com/kubernetes-engine/>`__, consider
   using the
   :ref:`GKEStartPodOperator <howto/operator:GKEStartPodOperator>` operator as it
```

Comment:
This link does currently appear to be working.  should it be updated?

Reply:
Some description that `GKEStartPodOperator` extends `KubernetesPodOperator` and that therefore the rest of this documentation is still applicable would be really helpful.

Reply:
This link works. However, it is not available in the source code, but works in the HTML code generated. Please look at: https://airflow.readthedocs.io/en/latest/howto/operator/kubernetes.html